pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -3721,7 +3721,8 @@
       ],
       "BaselineVersion": "4.5.1",
       "InboxOn": {
-        "uap10.0.16300": "4.0.5.0"
+        "uap10.0.16300": "4.0.5.0",
+        "netcoreapp3.0": "4.0.5.0"
```

Comment:
Does this require a version bump?

Reply:
No, it specifically should not get a version bump.  We want the OOB and inbox versions to be *interchageable* so that inbox will never be preferred over an OOB version with more API.

One could argue that this package has missed the opportunity for minor version bumps when it added API over the last stable release (our goal per semver) but that isn't something to address in this change.

Contrast that to System.Memory where the inbox version had more API than the OOB and we wanted to prefer the inbox *always*: that's what required an intentional version difference for inbox vs OOB.